packages/fluentui/accessibility-definitions/src/definitions/Button/buttonBehaviorDefinition.ts

```diff
@@ -0,0 +1,48 @@
+import { Rule, BehaviorRule } from '@fluentui/a11y-rules';
```

Comment:
I suggest to eliminate there a level of nesting: `fluentui/accessibility-definitions/src/definitions/Button/buttonBehaviorDefinition.ts` => `fluentui/accessibility-definitions/src/Button/buttonBehaviorDefinition.ts`

Reply:
I did this change